### ISSUE TITLE
Appium-Desktop Inspector replaces blank spaces in ID's and Names

### ISSUE THREAD

**Issue by VTINFOJames** [appium/appium-desktop#443]
## Appium Desktop

- [x] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

The Inspector will replace blank spaces in element Accessibility ID's and attributes (name, value) if there are more than one blank space between characters. 
I used the inspector to determine which elements had Accessibility ID's and which only had names. I then wrote a test that targeted those ID and Name values but the test would fail with a NoSuchElementException.
I then went back into the inspector and would copy the ID or Name from the inspector and search for the elements accordingly in the inspector and the elements were not found.
I then had my test build a list of elements by type and print out each element name as it iterated through them, and I found that a double space was being used in the names and ID's but the inspector would display them as a single space.
I reopened the inspector and searched for the elements using a double space and it was able to find them. It's difficult to see in the example below but the actual text has 2 spaces before and after the pipe symbol. 
Example element ID  #(actual):     
**Rep 312  |  Feb 6 '18**
Inspector displayed as: 
**Rep 312 | Feb 6 '18**

Please ensure the Inspector honors the actual text of the Accessibility ID or element attribute.

## Environment

- I am running Appium Desktop version v1.7.2.
- I am on (pick one):
    - [x] Mac - testing native iOS apps on a real device running iOS 11.2.6, xCode 9.2
    - [ ] Windows
    - [ ] Linux

**Comment by dpgraham** [comment]
@VTINFOJames How were you able to verify this? Did you make a call to `.getPageSource` and compared it to what's in the UI?

**Comment by Gilbert1024** [comment]
@VTINFOJames @dpgraham Same here. I have a label named "Sponsored.  XXXX" in my iOS app. And I can select it on Appium Desktop. However, it shows "Sponsored. XXXX", with ONE SPACE missed. 

Also, I try to print out page source using Appium. It shows the right label with TWO SPACE. And I can search and get that element on Appium Desktop. 

So for me, it looks like an Appium Desktop issue: combine multiple space(maybe tabs) into one.

**Comment by dpgraham** [comment]
It's probably Appium and not Appium Desktop that's reporting the label differently though.

**Comment by dpgraham** [comment]
Sorry for the late reply @Gilbert1024 

I don't know if a fix will be put in any time soon. What impact is this having on your tests? And is there a simple workaround for this issue?

**Comment by VTINFOJames** [comment]
Yes. @dpgraham. I ran getPageSource() and compared the output to the Appium-Desktop Inspector.

**Comment by Gilbert1024** [comment]
@dpgraham Thanks for replying. For me it is not really a big issue. Once I find some element is shown on Appium Desktop but not searchable, I just try to add/remove spaces/tabs in the search query.
